src/plugins/resolver/resolver.c

```diff
@@ -913,8 +913,6 @@ static void elektraUpdateFileTime (resolverHandle * pk, int fd, Key * parentKey)
 		ELEKTRA_ADD_WARNINGF (99, parentKey, "Could not update time stamp of \"%s\", because %s",
 				      fd == pk->fd ? pk->filename : pk->tempfile, strerror (errno));
 	}
-#else
-#warning futimens/futimes not defined
```

Comment:
Maybe this should even be an error.  The resolver plugin does not do what it promises if it cannot update file time.

If it is an error, the error that `#warning` is a GCC extension should be gone, too.

Reply:
even with the current implemntation i get warnings because apparently the function is undefined for the build.
No idea if that is a c preprocessor thing or if we really do not have the required libraries/settings.

Reply:
The resolver is something which almost certainly will never compile with c99. Use wresolver instead (see my cmake command).